pkg/cmd/server/origin/admission/chain_builder.go

```diff
@@ -98,7 +98,7 @@ var (
 		"Priority",
 		"ExtendedResourceToleration",
 		"DefaultTolerationSeconds",
-		"PVCProtection",
+		"StorageObjectInUseProtection",
```

Comment:
I normally avoid this during a rebase.  Why do we want to add this now?  This requires a paired finalization controller, right?

Reply:
Oh, it renamed?  that's mean.